cli/tsc/99_main_compiler.js

```diff
@@ -1034,6 +1032,13 @@ delete Object.prototype.__proto__;
           return respond(id, {});
         }
       }
+      case "$didChangeScripts": {
+        const specifiers = args[0];
+        for (const specifier of specifiers) {
+          scriptVersionCache.delete(specifier);
```

Comment:
What happens when an https file is changed on the file system? For example, when using `{ "vendor": true }` in a deno.json then editing something in the vendor folder?